test/models/work_test.rb

```diff
@@ -0,0 +1,153 @@
+require "test_helper"
+
+describe Work do
+
+  describe "validations" do
+
+    before do
+      @work = Work.create!(category: 'album', title: 'test title', creator: 'test creator', publication_year: 1999)
+    end
+
+    it "can be created with all required fields" do
+      result = @work.valid?
+
+      value(result).must_equal true
+    end
+
+    it "is invalid without a category" do
+      @work.category = nil
+
+      result = @work.valid?
+
+      value(result).must_equal false
+    end
+
+    it "is invalid without a title" do
+      @work.title = nil
+
+      result = @work.valid?
+
+      value(result).must_equal false
+    end
+
+    it "is invalid with a duplicate title and category" do
+      duplicate_work = Work.new(category: 'album', title: 'test title', creator: 'another creator', publication_year: 2000)
+
+      result = duplicate_work.valid?
+      value(result).must_equal false
+      duplicate_work.errors.messages.must_include :title, :creator
+    end
+
+    it "is valid with a duplicate title and different category" do
+      duplicate_work = Work.new(category: 'book', title: 'test title', creator: 'another creator', publication_year: 2000)
+
+      result = duplicate_work.valid?
+      value(result).must_equal true
+    end
+
+    it "is invalid without a creator" do
+      @work.creator = nil
+
+      result = @work.valid?
+
+      result.must_equal false
+    end
+
+    it "is invalid without a publication year" do
+      @work.publication_year = nil
+
+      result = @work.valid?
+
+      result.must_equal false
+    end
+
+  end
+
+  describe  "relationships" do
+
+    before do
+      @work = works(:album)
+    end
+
+    it "has many votes" do
+
+      @work.votes.each do |vote|
+        vote.must_be_kind_of Vote
+      end
+
+    end
+
+    it "has a list of voting users" do
+
+      @work.upvoted_users.each do |user|
+        user.must_be_kind_of User
+      end
+
+    end
+
+  end
+
+  describe "self.order_albums" do
+
+    it "returns a list of works with the category album" do
+      album = works(:album)
+      result = Work.order_albums
+
+      result.must_include album
+      result.length.must_equal 1
+      result.each do |work|
+        work.category.must_equal "album"
+      end
+    end
+  end
+
+  describe "self.order_books" do
+
+    it "returns a list of works with the category book" do
+      book = works(:book)
+      result = Work.order_books
+
+      result.must_include book
+      result.length.must_equal 1
+      result.each do |work|
+        work.category.must_equal "book"
+      end
+
+    end
+
+  end
+
+  describe "self.order_movies" do
+
+    it "returns a list of works with the category movie" do
+      movie = works(:movie)
+      result = Work.order_movies
+
+      result.must_include movie
+      result.length.must_equal 1
+      result.each do |work|
+        work.category.must_equal "movie"
+      end
+
+    end
+
+  end
+
+  describe "self.find_spotlight" do
+
```

Comment:
Like you said in your PR, there's definitely a lot of interesting test cases for your custom `Work` methods missing here. For `spotlight`, I would wonder:

- What happens if there are no works?
- What happens if there are works but no votes?
- What happens if two works have the same number of votes?

Similarly for you `order_x` methods, I would ask:

- What if there are no works of that category?